AIDriver.lua

```diff
@@ -196,7 +196,7 @@ end
 function AIDriver:beforeStart()
 	self.turnIsDriving = false
 	self.nextCourse = nil
-	self:deleteCollisionDetector()
+	-- self:deleteCollisionDetector()		-- makes no sense to delete the detector here
```

Comment:
Well, it does, as the idea is hide all details of the colli detector from the users like the AIDriver, so they don't have to care about any collision detector internals, only using the explicit interface of the TrafficCollision class (wish we used a proper language like Java, where the TrafficCollision class would have an interface with only constructor/destructor/update/getStatus)

Reply:
The colli detector will be initialized in AIDriver:init() when the AIDriver is activated as per my understanding.
When calling AIDriver:start() there will be called AIDriver:beforStart() and the TrafficCollision Detector then gets deleted.
Later it is created in AIDriver:detectCollision() if missing.
For me this makes not really sense.

Reply:
Oh, I see what you mean. Leave it as is.